expo/expo
# [create-expo-module] fix example app prebuild

# Why

Fixes #19552

# How

As reported suggested, we should use `npx` now to run prebuild in example app, which is setup by `create-expo-module`.

# Test Plan

* ensure that `expo` is not accessible in the test directory
* run updated `create-expo-module` bin and ensure that process completes

# Checklist
